### ISSUE TITLE
Changing max episode length.

### ISSUE THREAD

**Issue by ZhangTianrong** [chihyaoma/selfmonitoring-agent#17]
Hi, I was trying to train this model for R4R (an augmented version of R2R) which connects 2 instructions into 1. Therefore  I doubled the maximum episode length from 10 to 20 while leaving the rest of the hyperparameters unchanged. The training is then largely hindered, even the training loss isn't decreasing. I used to leave the maximum episode length as it was and the training was quite up to expectation in that setup. I am going to change the learning rate to a smaller value for now. Does anyone has more suggestions as to what needs to be further modified to account for a longer episode?

**Comment by ZhangTianrong** [comment]
I realised I perhaps is overacting since before I double the length, there was also a loss plateau at the beginning. The longer training time has probably made me too sensitive about this. I am closing this issue for now and wait for the training to proceed.
